app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.ProductService;
+import com.codesoom.assignment.domain.Product;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * 상품에 대한 요청을 받고 응답을 보낸다.
```

Comment:
잘 작성하셨어요. 만약 컨트롤러가 HTTP 프로토콜이 애플리케이션 내부로 침투하는 일을 막는다는 개념으로 생각한다면 이런 주석을 쓸 수도 있겠죠. 만약 컨트롤러가 HTTP 프로토콜의 침투를 막지 못한다면 URL, HTTP Method 같은 용어들이 서비스, 리포지토리까지 밀고 들어올 수 있을 테니까요.

```suggestion
 * 상품에 대한 HTTP 요청의 처리를 담당합니다.
```

Reply:
컨트롤러가 무엇을 하는지 의도 파악하기가 쉬워졌어요